src/aks-preview/HISTORY.rst

```diff
@@ -15,7 +15,7 @@ Pending
 19.0.0b15
 +++++++
 * Fix `NoneType` error when performing operations on automatic clusters that have hosted system components enabled.
-
+* `az aks bastion`: Correctly configure `$KUBECONFIG` values for tunneling traffic into a private AKS cluster.
```

Comment:
Please select a new version number and update setup.py to release a new version. Alternatively, you can move this to the "Pending" section for future release.

Reply:
Stuck it into pending, will let you all choose when & how to release.